patterns/template-home-news-blog.php

```diff
@@ -16,180 +16,127 @@
-						<!-- wp:query-no-results -->
-							<!-- wp:paragraph -->
-							<p><?php echo esc_html_x( 'Sorry, but nothing was found. Please try a search with different keywords.', 'Message explaining that there are no results returned from a search.', 'twentytwentyfive' ); ?></p>
-							<!-- /wp:paragraph -->
-						<!-- /wp:query-no-results -->
-					</div>
-					<!-- /wp:query -->
-					<!-- wp:query {"query":{"perPage":1,"pages":0,"offset":"4","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"taxQuery":null,"parents":[]}} -->
-					<div class="wp-block-query">
-						<!-- wp:post-template -->
-							<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"default"}} -->
-							<div class="wp-block-group">
-								<!-- wp:post-featured-image {"isLink":true,"aspectRatio":"3/2"} /-->
-								<!-- wp:post-title {"isLink":true,"fontSize":"large"} /-->
-								<!-- wp:post-terms {"term":"category","style":{"typography":{"textTransform":"uppercase","letterSpacing":"1.4px"}}} /-->
-							</div>
-							<!-- /wp:group -->
-						<!-- /wp:post-template -->
-						<!-- wp:query-no-results -->
-							<!-- wp:paragraph -->
-							<p><?php echo esc_html_x( 'Sorry, but nothing was found. Please try a search with different keywords.', 'Message explaining that there are no results returned from a search.', 'twentytwentyfive' ); ?></p>
-							<!-- /wp:paragraph -->
-						<!-- /wp:query-no-results -->
-					</div>
-					<!-- /wp:query -->
-				</div>
-				<!-- /wp:group -->
-			</div>
-			<!-- /wp:column -->
-		</div>
-		<!-- /wp:columns -->
-	</div>
-	<!-- /wp:group -->
+	<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|50"}}}} -->
+		<div class="wp-block-columns alignwide"><!-- wp:column {"width":"25%"} -->
+		<div class="wp-block-column" style="flex-basis:25%"><!-- wp:group {"style":{"layout":{"columnSpan":1,"rowSpan":1}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
+		<div class="wp-block-group"><!-- wp:query {"queryId":26,"query":{"perPage":1,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"taxQuery":null,"parents":[]}} -->
+		<div class="wp-block-query"><!-- wp:post-template -->
+		<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"default"}} -->
+		<div class="wp-block-group"><!-- wp:post-featured-image {"isLink":true,"aspectRatio":"3/2"} /-->
+
+		<!-- wp:post-title {"isLink":true,"fontSize":"large"} /-->
+
+		<!-- wp:post-terms {"term":"category","style":{"typography":{"textTransform":"uppercase","letterSpacing":"1.4px"}}} /--></div>
+		<!-- /wp:group -->
+		<!-- /wp:post-template -->
```

Comment:
Can we only remove the blocka? This adds query id's and removes the indentation, making the pattern code more difficult to read.